### PR TITLE
📖 Bento sample polish

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

File: extensions/amp-lightbox/1.0/example/index.html [hunk 1]
```diff
@@ -6,38 +6,59 @@
...
+        <h1>Lightboxed content</h1>
+        <button id="close-button">Close lightbox</button>
+      </div>
     </bento-lightbox>
     <button id="open-button">Open lightbox</button>
     <script>
       (async () => {
-        const lightbox = document.querySelector('#my-lightbox');
+        const lightboxElement = document.querySelector('#my-lightbox');
+        console.log('defined');
```
Comment (kvchari) [inline extensions/amp-lightbox/1.0/example/index.html:53]: remove log

File: extensions/amp-lightbox/1.0/example/index.html [hunk 2]
```diff
@@ -6,38 +6,59 @@
...
         await customElements.whenDefined('bento-lightbox');
-        const api = await lightbox.getApi();
+        console.log('defined');
```
Comment (kvchari) [inline extensions/amp-lightbox/1.0/example/index.html:55]: remove

**Review by kvchari — APPROVED** [review]
might want to remove logs

File: extensions/amp-lightbox/1.0/example/index.html (hunk 1 above)
Comment (sebastianbenz) [reply extensions/amp-lightbox/1.0/example/index.html]: done

File: extensions/amp-lightbox/1.0/example/index.html (hunk 2 above)
Comment (sebastianbenz) [reply extensions/amp-lightbox/1.0/example/index.html]: done

**Pull request merged**